scripts/launch.sh

```diff
@@ -1,23 +1,28 @@
 #!/usr/bin/env bash
+#
+# Helper scripts for automatically deploying WordPress plugins.
+#
+# https://github.com/wpsh/wpsh
+
 set -e
 
 echo "Deploying to plugin SVN."
 
 # Fetch SVN authentication data from the environment variables
 if [[ -z "$WP_ORG_USERNAME" || -z "$WP_ORG_PASSWORD" || -z "$WP_ORG_SLUG" ]]; then
-	echo "Missing WP.org username, password or slug."
+	echo "Missing WP.org username, password or slug." 1>&2
 	exit 1
 fi
 
-SVN_TAG=${1-"trunk"}
+SVN_TAG="${1-"trunk"}"
 BUILD_PATH="/tmp/plugin-build"
 SVN_PATH="/tmp/plugin-svn"
 GIT_PATH="$PWD"
 DEPLOY_MESSAGE="Deploy $SVN_TAG"
 
 echo "Git path: $GIT_PATH"
 
-if [ ! -d "$GIT_PATH/.git" ]; then
+if [ ! -d "$GIT_PATH/.git" ] || [ ! -f "$GIT_PATH/.git" ]; then
```

Comment:
It appears that git requires `.git` to be a directory. What is the idea behind checking for a `.git` file too?

Reply:
git-dir and work-tree may be in a different location as you may read it in `man git-init`
```
       --separate-git-dir=<git dir>
           Instead of initializing the repository as a directory to either $GIT_DIR or ./.git/, create a text file
           there containing the path to the actual repository. This file acts as filesystem-agnostic Git symbolic
           link to the repository.
```